.github/renovate.json

```diff
@@ -1,5 +1,6 @@
 {
 	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": ["config:js-lib"],
```

Comment:
I poked through the docs at https://docs.renovatebot.com/presets-config/#configjs-lib a bit but didn't see much info. Is there a reason to do this? I've been pretty happy with the config so far.

Reply:
These are the configs I'd personally recommend extending from:

```json
{
  "extends": [
    ":ignoreModulesAndTests",
    ":preserveSemverRanges",
    ":pinDevDependencies",
    ":widenPeerDependencies",
    "group:monorepos",
    "group:recommended",
    "workarounds:typesNodeVersioning"
  ]
}
```

Reply:
This is roughly the config I use if you are curious.

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    ":ignoreModulesAndTests",
    "group:monorepos",
    "group:recommended",
    ":automergeAll",
    ":automergePr",
    ":semanticCommits",
    ":prHourlyLimitNone",
    ":prConcurrentLimitNone",
    "helpers:disableTypesNodeMajor",
    "workarounds:typesNodeVersioning"
  ],
  "labels": ["Type: Maintenance", ":blue_heart:"],
  "automergeStrategy": "rebase",
  "packageRules": [
    {
      "matchManagers": ["npm"],
      "matchDepTypes": ["dependencies"],
      "rangeStrategy": "update-lockfile",
      "semanticCommitType": "chore",
      "semanticCommitScope": "deps",
      "major": {
        "semanticCommitType": "build"
      }
    },
    {
      "matchManagers": ["npm"],
      "matchDepTypes": ["devDependencies"],
      "rangeStrategy": "pin",
      "semanticCommitType": "chore",
      "semanticCommitScope": "dev-deps"
    },
    {
      "matchManagers": ["npm"],
      "matchDepTypes": ["peerDependencies"],
      "rangeStrategy": "widen",
      "semanticCommitType": "chore",
      "semanticCommitScope": "peer-deps",
      "major": {
        "semanticCommitType": "build"
      }
    },
    {
      "matchManagers": ["github-actions"],
      "rangeStrategy": "replace",
      "semanticCommitType": "ci",
      "semanticCommitScope": "dev-deps"
    }
  ]
}
```